notes/link-guidance.html

```diff
@@ -3,32 +3,36 @@
 <head>
 <meta charset='utf-8'/>
 <title>OSLC Core Version 3.0: Link Guidance</title>
-<script
-src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.24/builds/respec-oasis-common.min.js"
+<!-- <script
+src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.30/builds/respec-oasis-common.min.js"
 async
 class="remove"
-></script>
+></script> -->
+<script src="http://127.0.0.1:9000/respec-oasis-common.js" async class="remove"></script>
+
 <script class='remove'>
   var respecConfig = {
-      specStatus:           "PND", //PN Draft
+      shortName:            "link-guidance",
+      specStatus:           "PN", //PN Draft
```

Comment:
#558 makes changes to the same file. What's the difference?

Reply:
None, just an editing that happened in the wrong branch. Its hard to use different branches when I need to see similar changes I made on another branch that hasn't been merged to master yet.